cmd/minishift/cmd/config/view.go

```diff
@@ -74,7 +75,9 @@ func configView() error {
 		if excluded {
 			continue
 		}
-
+		if configViewFormat != DefaultConfigViewFormat {
+			configViewFormat = fmt.Sprintln(configViewFormat)
```

Comment:
Something is not quite right here. This feels like putting on a bandage on things.